civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -7475,4 +7460,4 @@ function get_civihr_version() {
   }
 
   return $civihrVersion;
-}
+}
```

Comment:
ah right, I did see the editorconfig file but I actually thought that this was for some people using vim or something :-D
I just installed it now